src/AppBundle/Service/ReportService.php

```diff
@@ -109,7 +111,6 @@ public function createNextYearReport(Report $oldReport)
         }
 
         $this->_em->persist($newReport);
-        $this->_em->flush();
```

Comment:
yeah, single transaction, better to fail than having the db dirty. Private method, so we have visibility on the calls